reference/docs-conceptual/getting-started/fundamental/Using-Variables-to-Store-Objects.md

```diff
@@ -1,30 +1,36 @@
 ---
-ms.date:  06/05/2017
+ms.date:  08/27/2018
 keywords:  powershell,cmdlet
 title:  Using Variables to Store Objects
 ms.assetid:  b1688d73-c173-491e-9ba6-6d0c1cc852de
 ---
 
-# Using Variables to Store Objects
-PowerShell works with objects. PowerShell lets you create variables, essentially named objects, to preserve output for later use. If you are used to working with variables in other shells remember that PowerShell variables are objects, not text.
+# Using variables to store objects
 
-Variables are always specified with the initial character $, and can include any alphanumeric characters or the underscore in their names.
+PowerShell works with objects. PowerShell lets you create named objects known as variables.
+Variables names can include the underscore character can any alphanumeric characters. When used in PowerShell, a variable is always specified using the \$ character followed by variable name.
+
+## Creating a variable
 
-### Creating a Variable
 You can create a variable by typing a valid variable name:
 
 ```
 PS> $loc
 PS>
 ```
 
-This returns no result because **$loc** does not have a value. You can create a variable and assign it a value in the same step. PowerShell only creates the variable if it does not exist; otherwise, it assigns the specified value to the existing variable. To store your current location in the variable **$loc**, type:
+This example returns no result because `$loc` doesn't have a value. You can create a variable and
+assign it a value in the same step. PowerShell only creates the variable if it doesn't exist.
+Otherwise, it assigns the specified value to the existing variable. The following example stores
+the current location in the variable `$loc`:
 
-```
+```powershell
 $loc = Get-Location
 ```
 
-There is no output displayed when you type this command because the output is sent to $loc. In PowerShell, displayed output is a side effect of the fact that data, which is not otherwise directed, always gets sent to the screen. Typing $loc will show your current location:
+No output displayed when you type this command because the output of `Get-Location` is sent to
+`$loc`. In PowerShell, data that isn't assigned or redirected is sent to the screen. Typing
+`$loc` shows your current location:
 
 ```
```

Comment:
PowerShell code-fence.

Reply:
The 1st sentence in the above is suboptimal (I hate passive voice).

How about: 
PowerShell displays no output when you type this command since PowerShell sends the output of 'Get-Location' to `$loc.

Reply:
I want to leave this as an *illustrative* example, per the style guide.

Reply:
@doctordns Thanks for the input. I made a change.